.changeset/empty-poems-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Fixed an issue where `SubNav` links and submenus didn't use sematically correct elements.
```

Comment:
Could we expand on this changelog entry with some more details please? Similar to what you did [here](https://github.com/primer/brand/pull/709/files#diff-f8904b87c5a588e5171d1c6f6aeec61741afc17ebcff12cf7f5e28b4b5ce04f4R5).

Reply:
Good shout, thanks. Lmk what you think of the new version